azure-functions-language-worker-protobuf/src/proto/identity/ClaimsIdentityRpc.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+// protobuf vscode extension: https://marketplace.visualstudio.com/items?itemName=zxh404.vscode-proto3
+
+// Light-weight representation of a .NET System.Security.Claims.ClaimsIdentity object.
+// This is the same serialization as found in EasyAuth, and needs to be kept in sync with
+// its ClaimsIdentitySlim definition, as seen in the WebJobs extension:
+// https://github.com/Azure/azure-webjobs-sdk-extensions/blob/dev/src/WebJobs.Extensions.Http/ClaimsIdentitySlim.cs
+message RpcClaimsIdentity {
```

Comment:
Are any of these values optional? The default values will be an empty string instead of `undefined`, and I don't know if that's the behavior you want or if it doesn't matter!

Reply:
After some verification, these properties can be optional. I'll update the protobuf!